src/rules/style.ts

```diff
@@ -69,6 +79,10 @@ export default {
   'semicolon': [true, 'always', 'ignore-interfaces'],
   // Require or disallow a space before function parenthesis
   'space-before-function-paren': spaceBeforeFunctionParen,
+  // Checks whether the final clause of a switch statement ends in break;.
+  'switch-final-break': true,
+  // Checks that type literal members are separated by semicolons. Enforces a trailing semicolon for multiline type literals.
+  'type-literal-delimiter': false,
```

Comment:
We have our own custom rule for this, [trailing-comma-interface](https://github.com/Shopify/tslint-config-shopify/blob/master/src/customRules/trailingCommaInterfaceRule.ts), so we have this covered.